integration-tests/__tests__/__snapshots__/expect_async_matcher.js.snap

```diff
@@ -0,0 +1,37 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`works with expected non promise values and not is a failure 1`] = `
```

Comment:
I'd like to see the stack trace as well here. does it point back to the correct code?

Reply:
Nop :(. When capturing the `error.stack` I get this in the snapshot:

```
exports[`works with expected non promise values is a failure 2`] = `
"Error: Expected value to have length:
  2
Received:
  1
received.length:
  1"
`;
```

If this is a different known issue, can we solve it in a different PR ? I'll be happy to tackle it.

Reply:
Yeah, #5104

Reply:
#6008